shared/metrics/src/main/java/io/pravega/shared/metrics/StatsLoggerImpl.java

```diff
@@ -77,100 +79,104 @@ public StatsLogger createScopeLogger(String scope) {
         if (0 == basename.length()) {
             scopeName = scope;
         } else {
-            scopeName = name(basename, scope);
+            scopeName = basename + "." + scope;
```

Comment:
Encapsulating the logic in a function is better as we don't have to go looking for instances to change in the case the format changes. I'm not sure why you are making this change here from calling `name` to formatting directly, but I'd suggest that you encapsulate the logic in a function. I'm also wondering whether that should be in `NameUtils`:

https://github.com/pravega/pravega/blob/679c7b7c35dd112977815de5abebc6e077ecc044/shared/protocol/src/main/java/io/pravega/shared/NameUtils.java

Reply:
The static "name" method is from Dropwizard library. 
Yes I'll clone this method into NameUtils.java.

Reply:
fixed